src/main/java/com/coherentsolutions/training/aqa/java/web/nikonova/browsers/BrowsersSettings.java

```diff
@@ -0,0 +1,107 @@
+            } else if (browser.equals(BrowsersProperties.getBrowserProperties("browser.firefox"))) {
+                firefoxOptions = new FirefoxOptions();
+                firefoxOptions.setPlatformName(BrowsersProperties.getBrowserProperties(String.format("%s.firefox.browser.platform", environment)));
+                firefoxOptions.setBrowserVersion(BrowsersProperties.getBrowserProperties(String.format("%s.firefox.browser.version", environment)));
+                driver = new RemoteWebDriver(new URL("http://localhost:4444/wd/hub"), firefoxOptions);
+            } else {
+                throw new Exception("Browser is unavailable");
+            }
+        } else if (environment.equals(BrowsersProperties.getBrowserProperties("env.sauceLabs"))) {
+            if (browser.equals(BrowsersProperties.getBrowserProperties("browser.chrome"))) {
+                driver = sauceLabsTestOnChrome();
+            } else if (browser.equals(BrowsersProperties.getBrowserProperties("browser.firefox"))) {
+                driver = sauceLabsTestOnFirefox();
+            } else {
+                throw new Exception("Browser is unavailable");
+            }
+        } else {
+            throw new Exception("Environment is unavailable");
+        }
+    }
+
+    public RemoteWebDriver sauceLabsTestOnFirefox() throws IOException {
+
+        String username = System.getenv("SAUCE_USERNAME");
+        String accessKey = System.getenv("SAUCE_ACCESS_KEY");
+        MutableCapabilities sauceOptions = new MutableCapabilities();
+
+        sauceOptions.setCapability("username", username);
+        sauceOptions.setCapability("accessKey", accessKey);
+
+        firefoxOptions = new FirefoxOptions();
+        firefoxOptions.setPlatformName(BrowsersProperties.getBrowserProperties(String.format("%s.firefox.browser.platform", environment)));
+        firefoxOptions.setBrowserVersion(BrowsersProperties.getBrowserProperties(String.format("%s.firefox.browser.version", environment)));
+
+        return new RemoteWebDriver(new URL("https://ondemand.us-west-1.saucelabs.com:443/wd/hub"), firefoxOptions);
+
+    }
+
+    public RemoteWebDriver sauceLabsTestOnChrome() throws IOException {
+
+        String username = System.getenv("SAUCE_USERNAME");
+        String accessKey = System.getenv("SAUCE_ACCESS_KEY");
+        MutableCapabilities sauceOptions = new MutableCapabilities();
+
+        sauceOptions.setCapability("username", username);
+        sauceOptions.setCapability("accessKey", accessKey);
+
+        chromeOptions = new ChromeOptions();
+        chromeOptions.setPlatformName(BrowsersProperties.getBrowserProperties(String.format("%s.chrome.browser.platform", environment)));
+        chromeOptions.setBrowserVersion(BrowsersProperties.getBrowserProperties(String.format("%s.chrome.browser.version", environment)));
+        return new RemoteWebDriver(new URL("https://ondemand.us-west-1.saucelabs.com:443/wd/hub"), chromeOptions);
+
+    }
+}
```

Comment:
I suggest to make these methods with private access modifier, because these methods will be used only in this class